### ISSUE TITLE
zooming problems on figma.com

### ISSUE THREAD

**Issue by shalva97** [minbrowser/min#1246]
* **Min Version:** 1.16.0
* **Operating system:** Arch Linux KDE

## Expected Behavior
figma.com handles all the zooming shortcuts that Min already has. so when i press "CTRL + =" it should be handled by Figma and not min. pressing CTRL + 0 should return to original zoom.

## Actual Behavior
if i press "CTRL + =" website gets zoomed twice, once by Min and then by the website. or if you zoom in by mouse and press CTRL + 0 it wont return to original zoom.

## To Reproduce
register on figma.com, create empty document, write a text to make zooming a bit more obvious. and then press ctrl + = and it will zoom too much.

**Comment by Akiyamka** [comment]
Same problem, also `ctrl + mouse wheel ` zoom browser ui with figma canvas =(

**Comment by PalmerAL** [comment]
The

**Comment by Akiyamka** [comment]
You are right, despite the fact that my problem fits the issue title, it problems of a different nature requiring different fixes. Maybe I should start a separate issue about preventDefault ignoring?

**Comment by PalmerAL** [comment]
@Akiyamka Your issue is fixed in the linked commit, it's actually the original issue here that still isn't fixed.

**Comment by Akiyamka** [comment]
@PalmerAL, aweasome, thx!
I build the latest version with this patch, figma now works fine!

**Comment by Syndamia** [comment]
> Your issue is fixed in the linked commit, it's actually the original issue here that still isn't fixed

**Comment by shalva97** [comment]
is there a way to somehow reset zoom level on Figma? accidently zoomed too much...

**Comment by PalmerAL** [comment]
I just had this same problem yesterday - menu > view > reset zoom fixes it.
